analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ExecutingSqlQueries.EF6.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Data.Entity;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Tests.Diagnostics
+{
+    class Program
+    {
+        public async Task DatabaseMethods(Database database, string query, int x)
+        {
+            database.SqlQuery<Program>(query + x); // Noncompliant {{Make sure using a dynamically formatted SQL query is safe here.}}
```

Comment:
Let's add one compliant case for each method name, to be sure it triggers based on the argument